src/components/menu/header-messages.jsx

```diff
@@ -53,7 +73,11 @@ export class HeaderMessagesUnconnected extends React.Component {
       case 'NEED_TO_LOGIN':
         content = (
           <span>
-            To save to this server, you need to <a onClick={this.props.login}>login</a>.
+            To save to this server, you need to
```

Comment:
I think you need to put this back on one line, webpack is jamming everything together with the line breaks:
![image](https://user-images.githubusercontent.com/20569/50565761-24331780-0d00-11e9-95fb-94a0dcc494d2.png)

(another alternative is to put an `&nbsp;` after `need to`, but I think it's better to just put things on one line)

Reply:
we can not put in one line because it is giving linting error line exceeds the maximum line length of 100.

Reply:
Ah I see, well then fall back to `&nbsp` I guess.